spdm.rflx

```diff
@@ -0,0 +1,1496 @@
+            then null;
+         Registry_ID : Registry_ID;
+         Vendor_ID_Len : Vendor_ID_Len
+            then Opaque_Error_Data
+               with Size => Message'Size - (Vendor_ID_Len'Last - Error_Code'First + 1)
+                  if Vendor_ID_Len = 0
+                     and Registry_ID = Reg_DMTF
+            then Vendor_ID_Double
+               if Vendor_ID_Len = 2
+                  and (Registry_ID = Reg_TCG
+                       or Registry_ID = Reg_USB
+                       or Registry_ID = Reg_PCI_SIG
+                       or Registry_ID = Reg_MIPI
+                       or Registry_ID = Reg_CXL
+                       or Registry_ID = Reg_JEDEC)
+            then Vendor_ID_Quad
+               if Vendor_ID_Len = 4
+                  and (Registry_ID = Reg_IANA
+                       or Registry_ID = Reg_HD_Base_T);
+         Vendor_ID_Double : Vendor_ID_Double
+            then Opaque_Error_Data
+               with Size => Message'Size - (Vendor_ID_Double'Last - Error_Code'First + 1);
+         Vendor_ID_Quad : Vendor_ID_Quad
+            then Opaque_Error_Data
+               with Size => Message'Size - (Vendor_ID_Quad'Last - Error_Code'First + 1);
+         Opaque_Error_Data : Opaque;
+      end message;
+
+   type Error_Response_1_1 is
+      message
+         Error_Code : Error_Code
+            then Invalid_Session_ID
+               if Error_Code = Invalid_Session
+            then Zero_Error_Data
+               if Error_Code = Invalid_Request
+                  or Error_Code = Busy
+                  or Error_Code = Unexpected_Request
+                  or Error_Code = Unspecified
+                  or Error_Code = Major_Version_Mismatch
+                  or Error_Code = Response_Not_Ready
+                  or Error_Code = Request_Resynch
+            then Reserved_Error_Data
+               if Error_Code = Decrypt_Error
+                  or Error_Code = Request_In_Flight
+                  or Error_Code = Invalid_Response_Code
+                  or Error_Code = Session_Limit_Exceeded
+            then Request_Response_Code
+               if Error_Code = Unsupported_Request
+            then Registry_ID
+               if Error_Code = Vendor;
+         Invalid_Session_ID : Error_Session_ID
+            then null;
+         Zero_Error_Data : Zero_Error_Data
+            then Response_Not_Ready_Data
+               if Error_Code = Response_Not_Ready
+            then null
+               if Error_Code /= Response_Not_Ready;
+         Reserved_Error_Data : Reserved_8
+            then null;
+         Response_Not_Ready_Data : Response_Not_Ready_Data
+            then null;
+         Request_Response_Code : Request_Code
+            then null;
+         Registry_ID : Registry_ID;
+         Vendor_ID_Len : Vendor_ID_Len
+            then Opaque_Error_Data
+               with Size => Message'Size - (Vendor_ID_Len'Last - Error_Code'First + 1)
+                  if Vendor_ID_Len = 0
```

Comment:
Wrong indentation.

Reply:
Fixed in https://github.com/Componolit/RecordFlux-specifications/pull/93/commits/4b2412a417abf4c2f782dc48e50069dea280daf2.